src/app/routes/utils/regex/utils/index.js

```diff
@@ -78,6 +78,11 @@ export const getOnDemandTvRegex = services => {
   return `/:service(${serviceRegex})/:serviceId(${tvMasterBrandRegex})/:brandEpisode(${brandEpisodeRegex})/:mediaId(${mediaIdRegex}):amp(${ampRegex})?`;
 };
 
+export const getTopicsPageRegex = services => {
+  const serviceRegex = getServiceRegex(services);
+  return `/:service(${serviceRegex})/topics/:id(${mediaIdRegex}):amp(${ampRegex})?`;
```

Comment:
Should probably be using a `assetUriRegex` here instead of media id
```suggestion
  return `/:service(${serviceRegex})/topics/:id(${assetUriRegex}):amp(${ampRegex})?`;
```
Or if the regex needs to be different, create a new one or rename media id

Reply:
Is this path accurate? Doesn't have to be perfect at this stage but would be a nice bonus for it to broadly reflect what we are going to build.

Reply:
> Should probably be using a `assetUriRegex` here instead of media id
> 
> Or if the regex needs to be different, create a new one or rename media id

Cool. I believe our intention is to create a new regex, we used the mediaIdRegix as a placeholder for now :)

Reply:
Path is accurate